VCFv4.3.tex

```diff
@@ -173,30 +173,49 @@ \subsubsection{Individual format field format}
 Possible Types for FORMAT fields are: Integer, Float, Character, and String (this field is otherwise defined precisely as the INFO field).
 
 \subsubsection{Alternative allele field format}
-Symbolic alternate alleles are described as follows:
+Symbolic alternate alleles specified in the ALT field are described as follows:
 \begin{verbatim}
 ##ALT=<ID=type,Description=description>
 \end{verbatim}
 
 \noindent \textbf{Structural Variants} \newline
-In symbolic alternate alleles for imprecise structural variants, the ID field indicates the type of structural variant, and can be a colon-separated list of types and subtypes.
-ID values are case sensitive strings and must not contain whitespace or angle brackets.
-The first level type must be one of the following:
-\begin{itemize}
-  \item DEL Deletion relative to the reference
-  \item INS Insertion of novel sequence relative to the reference
-  \item DUP Region of elevated copy number relative to the reference
-  \item INV Inversion of reference sequence
-  \item CNV Copy number variable region (may be both deletion and duplication)
-  \item BND Breakend
-\end{itemize}
 
-The CNV category should not be used when a more specific category can be applied. Reserved subtypes include:
-\begin{itemize}
-  \item DUP:TANDEM Tandem duplication
-  \item DEL:ME Deletion of mobile element relative to the reference
-  \item INS:ME Insertion of a mobile element relative to the reference
-\end{itemize}
+
+\begin{tabular}{l l}
+DEL  &  Deletion relative to the reference \\
+INS  &  Insertion relative to the reference \\
+DUP  &  Tandem duplication relative to the reference \\
```

Comment:
```suggestion
DUP  &  Duplication relative to the reference \\
```

As discussed, DUP is going to be just a generic “duplication” and the details will be provided in a separate SVCLAIM field #517.

Reply:
I miss that "duplication" refers to a "any quantitative increase of the number of alleles compared to the reference genome, without indication about the physical location of the additional copies". I.e. tandems can be a subset, as can be e.g. extrachromosomal amplifications.

Reply:
Agreed. That's an excellent phrasing as well, I've added it to the suggested change now